changelog.d/20220224_133151_jake_globus_collection.rst

```diff
@@ -0,0 +1,19 @@
+.. A new scriv changelog fragment.
+..
+.. Uncomment the header that is right (remove the leading dots).
+..
+.. Features
+.. --------
+..
+.. - A bullet item for the Features category.
+..
+.. Bugfixes
+.. --------
+..
+.. - A bullet item for the Bugfixes category.
+..
+ Documentation
+.. -------------
+..
+- Documentation and examples for the ``globus-collection`` input schema format.
+..
```

Comment:
"Documentation" has a leading space, and the next two lines still have the `dot-dot-space` prefix that makes them comments.

I think this won't get picked up correctly when the fragments are compiled together. It should match this format:

```rst
Documentation
-------------

- Documentation and examples for the ``globus-collection`` input schema format.
```

Reply:
Okay I think I got it this time, thanks for your help Kurt

Reply:
NP, thanks for all your work on the new features and the docs!!